protocols/virtual-fund/virtual-fund.go

```diff
@@ -61,7 +61,7 @@ func New(
 
 	var init VirtualFundObjective
 
-	if ledgerChannelToMyLeft.IsTwoPartyLedger && ledgerChannelToMyRight.IsTwoPartyLedger {
```

Comment:
Was this bug previously slipping though our unit tests? Should we make any changes to our unit tests?